vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -140,7 +140,7 @@
     </ClCompile>
     <Link>
       <AdditionalDependencies>winsqlite3.lib;ChakraRT.lib;dxguid.lib;dloadhelper.lib;OneCoreUap_apiset.lib;%(AdditionalDependencies)</AdditionalDependencies>
-      <DelayLoadDLLs>winsqlite3.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
+      <DelayLoadDLLs>chakra.dll;winsqlite3.dll;%(DelayLoadDLLs)</DelayLoadDLLs>
```

Comment:
is this intentional? Chakra != Hermes :)

Reply:
I think this is necessary to enable choosing Hermes at runtime (as opposed to at build time), which we need in order to have a single nuget binary

Reply:
@dannyvv this is to ensure that MSRN.dll won't try to load chakra.dll when RNHost is configured to use hermes.. <DelayLoadDLLs/> def to delay load hermes.dll is present in the ReactNative.Hermes.Windows.targets file in the hermes nuget package..